content/blog/release-0.1/index.md

```diff
@@ -0,0 +1,12 @@
+---
+title: "Announcing Release 0.1"
+date: "2019-12-20"
+---
+
+# Release 0.1 of the Coding Environment
+
+We’ve just released the first version of the [Coding Environment](https://build.codelearncreate.org/). This version features the ability to program the Wonder Workshop Dash robot to move forward, left, and right. We’ll build upon the Coding Environment adding new features with regular releases in the upcoming months. We’ve included a few activities in the Educator’s Toolkit for you to try. [Free play](https://resources.codelearncreate.org/lesson-plans-and-materials/dash-and-dot/free-play/),
```

Comment:
Were were calling it the "Inclusive Coding Environment?"

Maybe "This version features the ability to create programs that control the movements of Wonder Workshop's Dash robot"?